pkg/paho-mqtt/include/mqtt.h

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2019  Javier FILEIV <javier.fileiv@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+/**
+ * @ingroup     pkg_paho-mqtt
```

Comment:
Can you try

```suggestion
 * @addtogroup  pkg_paho-mqtt
```
 here? The content of this file does not [show up in the group documentation](https://139-7370241-gh.circle-artifacts.com/0/doc/group__pkg__paho-mqtt.html), only in the [header file's documentation](https://139-7370241-gh.circle-artifacts.com/0/doc/mqtt_8h.html). If that does not work, it is totally exceptable to move the content of `doc.txt` here and to remove that file:
```suggestion
 * @defgroup pkg_paho-mqtt   MQTT paho framework
 * @ingroup  pkg
 * @brief    The Eclipse Paho project provides open-source client implementations of MQTT for embedded systems
 * @see      https://github.com/eclipse/paho.mqtt.embedded-c
 *
 * The Eclipse Paho project provides open-source client
 * implementations of MQTT.
```

Reply:
Not sure if I got exactly what you wanted... could you please check? :)

Reply:
The problem is the `-` in the identifier. Changing it from `pkg_paho-mqtt` to `pkg_paho_mqtt` works as expected. For consistency, I would prefer to have `doc.txt` file as for all other packages which documents the package and groups the related files.